displays/parseCSV.py

```diff
@@ -0,0 +1,27 @@
+# Requirements: 
+# - Files must be in CSV formats. 
+# - Two CSV's must be entered into the function, with the first csv containing a "Digi-Key Part Number" column value and a "Mfg Std Lead Time" value. The second CSV should contain a "Mouser Part Number" column value and a "Lead Time in Weeks" value. 
+# - The third parameter is either the Digi-Key Part Number or the Mouser Part Number. The part number is to be entered as a string, as with the other CSV names. 
+
+
+import pandas as pd
+
+def parseCSV(csv1, csv2, partNumber):
+
+    df1 = pd.read_csv(csv1)
+    df2 = pd.read_csv(csv2)
+
+    try:
+        value = df1.loc[df1['Digi-Key Part Number'] == partNumber, "Mfg Std Lead Time"].values[0]
```

Comment:
+ Do we know for sure that there will only be one listing of the part number (ie. unique entry, and the filtering by `[0]` doesn't hide anything)?
+ Is the formatting for the part number consistent across both? (@mdshiozaki - do we have to parse around dashes or anything?)